components/brave_new_tab_ui/containers/newTab/settings/moreCards.tsx

```diff
@@ -189,6 +199,25 @@ class MoreCardsSettings extends React.PureComponent<Props, {}> {
           </StyledSettingsInfo>
           {this.renderToggleButton(showRewards, toggleShowRewards, false)}
         </SettingsWidget>
+        <FeaturedSettingsWidget>
+          <ToggleCardsWrapper>
+            <ToggleCardsText>
+              <ToggleCardsTitle>
+                {getLocale('cardsToggleTitle')}
+              </ToggleCardsTitle>
+              <ToggleCardsCopy>
+                {getLocale('cardsToggleDesc')}
+              </ToggleCardsCopy>
+            </ToggleCardsText>
+            <ToggleCardsSwitch>
+              <Toggle
+                size={'large'}
+                onChange={toggleCards.bind(this, !cardsHidden)}
```

Comment:
I think this should pass `cardsHidden` (instead of `!cardsHidden`) to toggleCards() method.
When cardsHidden is `true`, it means all cards are hidden and switch is off state.
If user clicks this switch, this switch will change from `off` to `on` and it means all cards should be visible.
So, `true` value should be used for `toggleCard` to show all cards.

Reply:
@simonhong yep, this was an oversight, thanks for catching, I've fixed this!